larray/inout/excel.py

```diff
@@ -197,7 +208,9 @@ def read_excel(filepath, sheet=0, nb_axes=None, index_col=None, fill_value=np.na
                             .format(list(kwargs.keys())[0]))
         from larray.inout.xw_excel import open_excel
         with open_excel(filepath) as wb:
-            return wb[sheet].load(index_col=index_col, fill_value=fill_value, sort_rows=sort_rows,
+            if range is None:
+                range = slice(None, None, None)
```

Comment:
* `range = slice(None)` is enough. 
* you could have `range` default to slice(None) directly since a slice object is readonly.